test/deferred-open-test.js

```diff
@@ -130,12 +132,12 @@ module.exports = function (test, testCommon) {
     })
   })
 
-  test('deferred open(): maxListeners warning', function (t) {
+  test('deferred open(): no maxListeners warning', function (t) {
     // 1) open database without callback, opens in next tick
     var db = testCommon.factory()
-    var stderrMock = sinon.mock(console)
+    var fail = t.fail.bind(t)
 
-    stderrMock.expects('error').never()
```

Comment:
Besides this not actually working anymore (because node emits a `warning` event rather than `error`), it caused problems when calling the test multiple times: sinon complained `console` was already wrapped.